### ISSUE TITLE
An EMF image does not render in DOC

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/metanorma-iso#746]
From https://github.com/metanorma/iso-19144-2/pull/9

An EMF image file is not rendering in DOC output though there are many
others of the same type that do render.

image file: [008.zip](https://github.com/metanorma/metanorma-iso/files/9135625/008.zip)

Markup:
```adoc

[[scls_8-8]]
=== LC_VegetationElement

==== LC_VegetationElement subtypes

The LC_VegetationElement metalanguage object is a subtype of LC_Element.
It has one subtype: LC_GrowthForm and has a relationship
_growthFormCharacteristics_ to a component LC_LandCoverElementCharacteristic.
This is represented in <<figure_7>>.

[[figure_7]]
.Subtypes of LC_VegetationElement
image::008.emf["",665,374]
````

DOC output:
![issue7-1](https://user-images.githubusercontent.com/33627611/179613499-3cab29c9-f9d4-4c15-93c0-86c053503242.PNG)

HTML output:
![issue7-2](https://user-images.githubusercontent.com/33627611/179613522-ee535859-8d76-4721-99c7-2ebe0a3f0d8f.PNG)

**Comment by ronaldtse** [comment]
I was able to open this in Adobe Illustrator but nonetheless it looks defective.

<img width="801" alt="Screenshot 2022-07-19 at 1 40 53 PM" src="https://user-images.githubusercontent.com/11865/179666104-73b8d301-9552-42db-839f-b7e84a53d297.png">

I think this may be an issue with https://github.com/metanorma/libemf2svg . Can you post the issue there?

CC: @maxirmx

**Comment by ronaldtse** [comment]
Actually, `emf2svg` was able to convert it properly. Here it is.
![x](https://user-images.githubusercontent.com/11865/179666723-41324723-c443-4880-ba88-800e4293cf25.svg)

Maybe there is some error in Metanorma that prevented it from working?

**Comment by ronaldtse** [comment]
Or maybe this is a DOC compatibility issue with this particular EMF?

**Comment by ronaldtse** [comment]
@manuelfuenmayor the EMF image shows properly in my generated Word document:
```adoc
==== LC_VegetationElement subtypes

The LC_VegetationElement metalanguage object is a subtype of LC_Element.
It has one subtype: LC_GrowthForm and has a relationship
_growthFormCharacteristics_ to a component LC_LandCoverElementCharacteristic.
This is represented in <<figure_7>>.

[[figure_7]]
.Subtypes of LC_VegetationElement
image::008.emf["",665,374]
```

<img width="967" alt="Screenshot 2022-07-19 at 10 04 06 PM" src="https://user-images.githubusercontent.com/11865/179756848-422196a6-e301-4e82-b7b5-265ffddfe736.png">

Can you upgrade Metanorma and see if it fixes the poblem for you?

**Comment by manuelfuenmayor** [comment]
@ronaldtse, no. I get the same result. I've updated my gems and tried compiling just the image along (with little content) and with other flavors (like OGC), and nothing.
@anermina can you try to generate this image in DOC output, please?

**Comment by maxirmx** [comment]
> I was able to open this in Adobe Illustrator but nonetheless it looks defective.
> 
> <img alt="Screenshot 2022-07-19 at 1 40 53 PM" width="801" src="https://user-images.githubusercontent.com/11865/179666104-73b8d301-9552-42db-839f-b7e84a53d297.png">
> 
> I think this may be an issue with https://github.com/metanorma/libemf2svg . Can you post the issue there?
> 
> CC: @maxirmx

@ronaldtse 
If you mean missing lines, these are actually lines that are scaled down to zero width by the applciation you use.
I am not sure if it can be fixed in https://github.com/metanorma/libemf2svg but in any case it is not related to this issue (IMHO)

**Comment by maxirmx** [comment]
@manuelfuenmayor 
Can you attach the broken DOC file please ?   If it is already attached, I am sorry. Pls point me to it.

**Comment by manuelfuenmayor** [comment]
@maxirmx
Sure, here you are:  [document.zip](https://github.com/metanorma/metanorma-iso/files/9143180/document.zip)

**Comment by anermina** [comment]
> @anermina can you try to generate this image in DOC output, please?

Just to confirm that I am also not able to generate this image when using Windows OS.

**Comment by ronaldtse** [comment]
@maxirmx 
> in any case it is not related to this issue (IMHO)

Indeed -- emf2svg was able to convert the image on my Apple M1 without issue. Thanks!

Perhaps this is only a Windows issue.

@CAMOBAP could you have a look whether you can:
1. See the 008.emf on Windows
2. Run the command `emf2svg 008.emf x.svg` on Windows and see how x.svg looks like?

Thanks!
